scripts/criu-ns

```diff
@@ -139,6 +155,23 @@ def wrap_restore():
         _mount_new_proc()
         run_criu(restore_args)
 
+    if restore_pidfile:
+        restored_pid = None
+        retry = 5
+
+        while not restored_pid and retry:
+            with open('/proc/%d/task/%d/children' % (criu_pid, criu_pid)) as f:
+                line = f.readline().strip()
+                if len(line):
+                    restored_pid = line
+                    break
+            retry -= 1
+            time.sleep(1)
```

Comment:
@Snorch Would it make sense to show a warning or error message in the case of timeout?